src/api/_mocks/characters/get/mock.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import fetchMock from 'fetch-mock';
+/* eslint-disable import/prefer-default-export */
```

Comment:
Regarding these two linter silencers:

- For the `no-extraneous-dependencies`, how do you feel about instead configuring ESLint to allow the usage of `devDependencies` when the file is inside the `_mocks` folder? See [the documentation here](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md#options) (specifically the part about globbing) for an example.
- For `prefer-default-export`, do we need it at all? It seems like you're already exporting the function as a default export.

Reply:
I've added the an override in the eslint file to disable this the `no-extraneous-dependencies` rule for all `.ts` files under `_mock` folder. The `prefer-default-export` rule was just wrong on my part I guess.